ustriage/ustriage.py

```diff
@@ -754,6 +756,13 @@ def launch():
                         dest='extended_format',
                         help='Do Display bugs in extended format which adds'
                              ' date-last-updated, importance and assignee')
+    parser.add_argument('-A', '--age',
```

Comment:
I find `--age` a bit confusing, it reminds me of the age of the bug (how long ago the bug was *filed*). What about something like `--flag-recent`? But let me know if I misinterpreted anything here.

Reply:
I'm ok with your suggested option name, adapted